pages/en/lb4/Command-line-interface.md

```diff
@@ -0,0 +1,25 @@
+---
+lang: en
+title: 'Command-line interface'
+keywords: LoopBack 4.0, LoopBack 4
+tags:
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Command-line-interface.html
+summary:
+---
+
+LoopBack 4 provides command-line tools to effortlessly scaffold applications from the ground up. LoopBack 4's CLI is equipped with generators to scaffold both applications and extensions as well as subgenerators for their artifacts such as Controllers and Providers. Once generated, the scaffold can be expanded with users' own code as needed.
```

Comment:
1. Split this into multiple lines to make it easier to review (it also helps keep the diffs small for subsequent PRs)
2. Avoid the word "effortlessly"; it's the sort of thing that people will harp on in the future if they decide that what we've given them isn't effortless.
3. I'd also avoid mentioning things that we haven't yet created, since there's no guarantees around _what_ we intend to generate for users, or how we'll do it.
`
LoopBack 4 provides command-line tools to help you get started quickly, by generating application and extension projects and installing their dependencies for you.
`

Reply:
+1  for point3.